single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DiagramStyles.java

```diff
@@ -36,6 +36,8 @@
     public static final String TOP_FEEDER = STYLE_PREFIX + "top-feeder";
     public static final String FRAME_CLASS = STYLE_PREFIX + "frame";
     public static final String NODE_INFOS = STYLE_PREFIX + "node-infos";
+    public static final String POWERED_CLASS = STYLE_PREFIX + "powered";
+    public static final String UNPOWERED_CLASS = STYLE_PREFIX + "unpowered";
```

Comment:
Following what I said before, those two classes should be in the components.css of the test, one for each component.